### PR TITLE
Adjustments to report formatting, update to treat last logged in like…

### PR DESCRIPTION
**Pull request by jules0x**
… standard user report does

### CODE REVIEW

File: src/Reports/LockedUsersReport.php [hunk 1]
```diff
@@ -55,7 +57,17 @@ public function columns()
             'Email' => 'Email',
             'Locked' => 'Locked',
             'getGroupNames' => 'Groups',
-            'getLastAccessed' => 'Last Logged In',
+            'Date' => [
+                'title' => 'Last logged in',
+                'formatting' => function ($value, $item) {
+                    $customDate = $item->getLastLogin();
+                    $item->shouldBeLockedOut();
```
Comment (alex-dna) [inline src/Reports/LockedUsersReport.php]: That doesn't do anything does it?

File: src/Extensions/MemberLockoutExtension.php [hunk 2]
```diff
@@ -57,29 +58,50 @@ public function getLockIfInactiveAfter() : DBDateTime
...
+        )->min('LockoutThresholdDays') ?: $defaultLockoutThreshold;
 
         return DBDateTime::now()->modify("-{$lowestThreshold} days");
     }
 
     /**
      * Return the date the user has last accessed the CMS
      */
-    public function getLastAccessed() : DBDatetime
+    public function getLastAccessed()
```
Comment (michalkleiner) [inline src/Extensions/MemberLockoutExtension.php]: No need to remove the type
```suggestion
    public function getLastAccessed(): ?DBDateTime
```

File: src/Extensions/MemberLockoutExtension.php [hunk 3]
```diff
@@ -57,29 +58,50 @@ public function getLockIfInactiveAfter() : DBDateTime
...
+    }
 
-        // Check for LoginSessions, and overwrite the default "Last" value
-        if ($this->owner->LoginSessions()->exists()) {
-            $latestLoginSession = $this->owner->LoginSessions()->sort('LastAccessed', 'DESC')->first();
-            $lastAccessed = $latestLoginSession->dbObject('LastAccessed');
+    /**
+     * Get the last login attempt
+     *
+     * @return void
```
Comment (michalkleiner) [inline src/Extensions/MemberLockoutExtension.php]: Incorrect type

File: src/Extensions/MemberLockoutExtension.php [hunk 4]
```diff
@@ -57,29 +58,50 @@ public function getLockIfInactiveAfter() : DBDateTime
...
+     */
+    public function getLastLogin()
```
Comment (michalkleiner) [inline src/Extensions/MemberLockoutExtension.php]:
```suggestion
    public function getLastLogin(): ?DBDateTime
```

File: src/Reports/LockedUsersReport.php (hunk 1 above)
Comment (michalkleiner) [reply src/Reports/LockedUsersReport.php]: Yeah, looks like this can either be removed or maybe needs to be moved above the `getLastLogin` call

Comment (jules0x) [reply src/Reports/LockedUsersReport.php]: Sorry that was a test

**Pull request merged**